x-pack/plugins/cases/server/saved_object_types/import_export/export.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CoreSetup,
+  Logger,
+  SavedObject,
+  SavedObjectsClientContract,
+  SavedObjectsExportTransformContext,
+} from 'kibana/server';
+import {
+  CaseUserActionAttributes,
+  CASE_COMMENT_SAVED_OBJECT,
+  CASE_SAVED_OBJECT,
+  CASE_USER_ACTION_SAVED_OBJECT,
+  CommentAttributes,
+  MAX_DOCS_PER_PAGE,
+  SAVED_OBJECT_TYPES,
+} from '../../../common';
+import { createCaseError, defaultSortField } from '../../common';
+import { ESCaseAttributes } from '../../services/cases/types';
+
+export async function handleExport({
```

Comment:
We intentionally do not want to do authorization checks while exporting because it is valid for a user to export a case while not having either security solution or observability cases read privileges.

Reply:
Why is valid? It seems like users can always read a case this way even if they do not have read access.

Reply:
Christos and I talked offline. This is the current behavior with other entities within Kibana. We don't have a way to enforce the feature privileges for import/export right now.